extensions/amp-nexxtv-player/0.1/test/validator-amp-nexxtv-player.html

```diff
@@ -0,0 +1,58 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Tests for the amp-nexxtv-player tag.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="./regular-html-version.html" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <!--<script async src="https://cdn.ampproject.org/v0.js"></script>-->
+    <script async custom-element="amp-nexxtv-player" src="/dist/v0/amp-nexxtv-player-0.1.max.js"></script>
+    <script async src="/dist/amp.js"></script>
+</head>
+<body>
+
+    <amp-nexxtv-player
+        data-mediaid="PTPFEC4U184674"
+        data-client="583"
+        data-streamtype="video"
+        data-seek-to="2"
+        data-mode="static"
+        data-origin="https://embed.nexx.cloud/"
+        layout="responsive"
+        width="480" height="270"
+    ></amp-nexxtv-player>
+
+
+<hr>
+    <amp-nexxtv-player
+            data-mediaid="PTPFEC4U184674"
```

Comment:
please use the `autoplay` attribute for one of these examples. Please test manually in iOS and Android to ensure `autoplay` works ( if you run into issue on iOS, your player may required `playsinline` attribute as iOS requires that on the `<video>` tag ). Other issues I have seen preventing `autoplay` is `mute` function setting volume to 0 instead of setting `muted` attribute